client/my-sites/upgrades/controller.jsx

```diff
@@ -178,6 +179,8 @@ module.exports = {
 				<CheckoutData>
 					<Checkout
 						cards={ storedCards }
+						plan={ context.params.plan }
```

Comment:
Maybe use `planName` or `selectedPlanName` instead of `plan` to show implicitly that it holds a string with the short name for the plan.